src/sentry/receivers/onboarding.py

```diff
@@ -20,6 +24,23 @@
 from sentry.utils.javascript import has_sourcemap
 
 
+def check_for_onboarding_complete(organization):
```

Comment:
im still not entirely sure where you see this getting called over and over? everything im looking at only does it when rows are affected